<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newExperimentDialog</class>
 <widget class="QDialog" name="newExperimentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="bbOKCancel">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>550</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>New Experiment </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tfExperimentName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>481</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>611</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>Add components: </string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>591</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tfComponentName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Host:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tfComponentHost">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tfComponentPort">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Subsystem:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tfComponentSubsystem">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="bAddComponentToList">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>10</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bRemoveComponentFromList">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>40</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableComponents">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>591</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>591</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Add components in file:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tfComponentListFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>421</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="bBrowseComponentList">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>84</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bAddComponentListFile">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>51</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>100</y>
     <width>351</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>Add configurations:</string>
   </property>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>331</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tfConfigurationName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>251</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="bAddConfigurationToList">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bRemoveConfigurationFromList">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableConfigurations">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>331</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>119</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Add configurations in folder:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tfConfigurationFolder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="bAddConfigurationFolder">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>51</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bBrowseConfigurationFolder">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>84</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bbOKCancel</sender>
   <signal>accepted()</signal>
   <receiver>newExperimentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bbOKCancel</sender>
   <signal>rejected()</signal>
   <receiver>newExperimentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
